compiler/rustc_hir/src/arena.rs

```diff
@@ -29,6 +29,7 @@ macro_rules! arena_types {
             [] field_pat: rustc_hir::FieldPat<$tcx>,
             [] fn_decl: rustc_hir::FnDecl<$tcx>,
             [] foreign_item: rustc_hir::ForeignItem<$tcx>,
+            [] foreign_item_ref: rustc_hir::ForeignItemRef<$tcx>,
```

Comment:
would it make sense to use `[few]` here?

Don't know how big of an impact this has but foreign items are probably quite rare.